tests/data_context/test_data_context_datasources.py

```diff
@@ -453,3 +453,21 @@ def test_list_datasources() -> None:
     context.sources.add_pandas(datasource_name)
 
     assert len(context.list_datasources()) == 2
+
+
+@pytest.mark.unit
+def test_get_available_data_assets_names(test_df_pandas, empty_data_context) -> None:
+
+    datasource_name = "my_fluent_pandas_datasource"
+    datasource = empty_data_context.sources.add_pandas(datasource_name)
+    datasource.add_dataframe_asset(name="test_data_frame", dataframe=test_df_pandas)
+
+    assert len(empty_data_context.get_available_data_asset_names()) == 1
+
+    data_asset_names = dict(
+        empty_data_context.get_available_data_asset_names(
+            datasource_names=datasource_name
+        )
+    )
+
+    assert "test_data_frame" in str(data_asset_names.values())
```

Comment:
Nit - instead of using two separate strings, can we do something like this:

```
asset_name = "test_data_frame"
...
datasource.add_dataframe_asset(name=asset_name)
...
assert asset_name in data_asset_names.values()
```

Reply:
also this casting to a string on L473 shouldn't be necessary